### DISCUSSION TITLE
"Maybe" option

### DISCUSSION THREAD

**Discussion by shaddocatto** [lukevella/rallly#147] [Ideas]
It would be nice to have the ability to Mark a date as tentative or maybe. Perhaps it could be a yellow check mark instead of green if you click the box twice. Multiple users for my polls have requested it.

**Comment by erikogan** [comment]
I agree that having a Maybe (better, to me, would be “If Need Be”) option would be a very welcome addition. It’s one of the few reasons I cannot convince our group to switch from Doodle.

@lukevella I’m debating diving in and seeing if I can offer this as a PR. Here’s my current thinking: make it optional (and not enabled by default) on event creation. If enabled, when selecting dates the first click on a checkbox becomes “Yes” (indicated with a green background?), the second becomes “Maybe / If Need Be” (yellow background?) and third deselects.

Would you object to that? I don’t know how invasive what I’m proposing might be, and I’d hate to do the work if it won’t be accepted.

**Comment by jaredatron** [comment]
👍 yes please

**Comment by hairykiwi** [comment]
@erikogan I agree with your proposed cyclical-style interface.

Whether titled "Maybe" or "If Need Be" could even be configured by the event organiser - or at least consider adding both options. By comparison, WhenAvailable.com's 3-option interface is a slow UI experience for users marking up more than a few days.

**Reply by ymyke** [reply]
Strong +1 for Ifneedbe rather than Maybe. Because Ifneedbe has clear semantics, it's a "yes, but", which is still helpful for the organizer. In contrast, a Maybe doesn't help much to decide whether an option is good or not.

**Reply by sandstrom** [reply]
I agree with this!

**Comment by Gabor111a** [comment]
+1 definitely!

@lukevella thank you for your awesome job!

**Comment by lukevella** [comment]
Hey folks, this is live now. Let me know what you think.

https://user-images.githubusercontent.com/676849/168238641-844bfa5a-3f11-4ebd-9772-a2373525dcd9.mp4

.

**Reply by Gabor111a** [reply]
Seems perfect!
Just one suggestion in the legend, for better readability, spaces should be realigned like this:
![kép](https://user-images.githubusercontent.com/79718906/168246641-b6679e0e-14b9-4a16-8bc5-29fa5f9bf080.png)

**Reply by lukevella** [reply]
Fair point 👍

**Reply by sandstrom** [reply]
Awesome! 🏅

**Reply by sean-wils** [reply]
Very happy with this!
